docs_2.0/manual_EN/5.deployment-and-installation/1.resource-preparations.md

```diff
@@ -0,0 +1,181 @@
+
+> **WARNING**: Do not deploy a cluster across IDCs.
+
+| Machine name | Number of metad | Number of storaged | Number of graphd |
+| ------------ | --------------- | ------------------ | ---------------- |
+| A            | 1               | 1                  | 1                |
+| B            | 1               | 1                  | 1                |
+| C            | 1               | 1                  | 1                |
+| D            | None            | 1                  | 1                |
+| E            | None            | 1                  | 1                |
+
+## Reserve resources for running a Nebula Graph cluster
+
+You can estimate the memory, storage, and partition number needed for a Nebula Graph cluster of 3 replicas as follows.
+
+| Resource                               | How to estimate                                                                                                                                                                |
+| -------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| Storage for a cluster                  | _the sum of edge number and vertex number_ * _average bytes of attributes_ * 6 * 120%                                                                                          |
+| Memory for a cluster                   | [_the sum of edge number and vertex number_ * 15 bytes + _the number of RocksDB instances_ * (_write_buffer_size_ * _max_write_buffer_number_) + _rocksdb_block_cache_] * 120% |
+| Number of partitions for a graph space | _the number of disks in the cluster_ * _disk_performance_index_                                                                                                                |
+
+* Question 1: Why do we multiply the storage and memory by 120%?
+
+    Answer: The extra 20% is for buffer.
+
+* Question 2: How to get the number of RocksDB instances?
+
+    Answer: Each directory in the `--data_path` item in the `etc/nebula-storaged.conf` file corresponds to a RocksDB instance. Count the number of directories to get the RocksDB instance number.
+
+    > **NOTE**: You can decrease the memory size occupied by the bloom filter by adding `--enable_partitioned_index_filter=true` in `etc/nebula-storaged.conf`.
+
+* Question 3: What is the _disk_performance_index_?
+
+    Answer: _disk_performance_index_ is a value between 2 to 10, the better performance of the hard disk, the larger the value. Use 2 for HDD.
+
+## Optimize the configuration for HDD and Gigabit Networks
+
+[This part might be moved to the configuration doc map later.]
+
+Nebula Graph is intended for NVMe SSD and 10 Gigabit Networks. Optimizing the configuration as follows may better accommodate HDD and Gigabit Networks.
```

Comment:
Nebula Graph is intended for NVMe SSD. 

> **It is not recommended to run on HDD!**. 

Try the configuration as follows may better accommodate HDD .